frontend/src/components/PointPopup/IconsComponent/IconsComponent.tsx

```diff
@@ -131,7 +133,7 @@ export const IconsComponent: FC<Props> = ({
           role === 'moderator' ||
           role === 'admin') && (
           <MenuItem onClick={handleClose}>
-            <ListItemIcon onClick={() => null}>
+            <ListItemIcon onClick={handleDeleteClick}>
               <DeleteIcon fontSize="small" />
             </ListItemIcon>
```

Comment:
you should use example in IconsCopmonent.tsx for edit location for you code - because delete logic work only on icon

Reply:
done